Dockerfile-linux.template

```
@@ -283,6 +283,8 @@ end
 	find /usr/local -type f -executable -not \( -name '*tkinter*' \) -exec ldd '{}' ';' \
 		| awk '/=>/ { print $(NF-1) }' \
 		| sort -u \
+		| awk '{ print $1; system("realpath " $1) }' \
+		| sort -u \
```

Some light "awk(ward) golf" :sweat_smile:

This essentially takes paths like `/lib/foo.so` and `/usr/lib/foo.so` and normalizes _both_ to `lib/foo.so`, which is more likely to match accurately in `dpkg-query --search` no matter which path the `.deb` happened to record.  I also figured it's a good opportunity to make sure we ignore `/usr/local` entries :smile:

```suggestion
		| awk '/=>/ { so = $(NF-1); if (index(so, "/usr/local/") == 1) { next }; gsub("^/(usr/)?", "", so); print so }' \
		| sort -u \
```

(I'm happy to take over from here if you'd prefer, but I want to give you first crack since you've done all the work so far :+1:)